client/db/bolt/db.go

```diff
@@ -1844,6 +1844,16 @@ func (db *BoltDB) NotificationsN(n int) ([]*dexdb.Notification, error) {
 			}
 			note.Ack = bEqual(trio.b.Get(ackKey), byteTrue)
 			note.Id = note.ID()
+			if !bytes.Equal(note.Id, trio.k) {
+				if !note.Ack {
+					// This notification was initially stored when the serialization
+					// and thus note ID did not include the TopicID. Ignore it.
+					db.log.Tracef("Ignoring stored note with bad key: %x != %x \"%s\"",
+						[]byte(note.Id), trio.k, note.String())
+					// trio.b.Put(ackKey, byteTrue)
```

Comment:
We could even move it to a new bucket with the correct key, but these are 1-1.5 year old notifications and I kinda don't wanna bother.

Reply:
I'm good with auto-acking these.

Reply:
Acking.  The two unacknowledgeable notes logged once, and are now gone after restart.  I think this should be good for this fairly rare situation (upgraded with unacked notes, over a year ago).

Reply:
Also worth noting that these notes would fall off the latest 100 list eventually anyway, but that can take a while for light use.

Reply:
Auto-acking in last force push.